autosubmitconfigparser/config/configcommon.py

```diff
@@ -626,7 +626,7 @@ def _normalize_wallclock(data_fixed: {}) -> None:
         """
         for job in data_fixed.get("JOBS", {}):
             wallclock = data_fixed["JOBS"][job].get("WALLCLOCK", "")
-            if wallclock and re.match(r'^\d{2}:\d{2}:\d{2}$', wallclock):
+            if wallclock and re.match(r'^\\d{2}:\\d{2}:\\d{2}$', wallclock):
```

Comment:
As it's a raw string, shouldn't it be `\d`? I think we want to match two digits, colon, two digits, colon, two digits.